CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - Removed location update in background
 - Fixed a crash on receiving location updates while stopping `VisionManager`
 - Fixed a crash due to race condition in `ObservableVideoSource`
+- Fixed freeze on init
```

Comment:
The freeze was introduced in this release so. No need to mention this.

Reply:
will remove

Reply:
removed